src/main/webapp/app/app.constants.ts

```diff
@@ -12,4 +12,6 @@ export const MIN_POINTS_ORANGE = 40;
 
 export const TUM_REGEX = /^([a-z]{2}\d{2}[a-z]{3})$/;
 
+export const CUSTOM_USER_KEY = 'customUserLogin';
```

Comment:
So here it is different then on the server:
`otherUserLogin` vs. `customUserLogin`.
I think it's most important that the naming is consistant.